client/less/lib/bootstrap/modals.less

```diff
@@ -90,6 +90,7 @@
 .modal-title {
   margin: 0;
   line-height: @modal-title-line-height;
+  color: #f9f9f9;
```

Comment:
Use the bootstrap variables instead of hardcoding hex values. I'm sure we have a variable already for this color.